sql_generators/glean_usage/templates/dau_reporting_clients_first_seen_v1.query.sql

```diff
@@ -0,0 +1,63 @@
+{{ header }}
+
+WITH
+  _current AS (
+    SELECT
+      client_info.client_id,
```

Comment:
if client_id is going away and swapped with usage_profile_id, we may need to change the bases of our aggregation to usage_profile_id. This could also impact the first_seen computation

Reply:
yes, we can do that once we actually start receiving `usage_profile_id` the issue is that the field does not exist at the moment.